### PR TITLE
completed sql-join exercise.

### PR DESCRIPTION
**Pull request by joseph-w-valdez**
image

### CODE REVIEW

File: sql-join/what-and-when.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/*
+Use two joins to select the "releaseYear" and "genres"."name" of the
+film with the "title" 'Boogie Amelie'. There is a special table in the
+database named "filmGenre" for this purpose.
+ */
+
+SELECT "line1", "cities"."name" as "city", "district", "countries"."name" as "country"
```
Comment (RobertGardner) [inline sql-join/what-and-when.sql:7]: Nice use of aliases.

File: sql-join/where-are-they.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+ /*
+ SELECT "line1", "cities"."name" as "city", "district"
+ from "addresses"
+JOIN "cities" using ("cityId")
+  */
+
+SELECT "line1", "cities"."name" as "city", "district"
+ from "addresses"
+JOIN "cities" using ("cityId")
```
Comment (RobertGardner) [inline sql-join/where-are-they.sql:15]: No country name. ;-)

File: sql-join/what-and-when.sql (hunk 1 above)
Comment (RobertGardner) [inline sql-join/what-and-when.sql:7]: This seems to be solving the wrong problem. Did you get mixed up? It looks like you may have accidentally overwritten this file when you added country name to `where-are-they.sql`.

**Review by RobertGardner — CHANGES_REQUESTED** [review]
magic-mallrats.sql is empty and it looks like you lost a solution. :-(

File: sql-join/what-and-when.sql (hunk 1 above)
Comment (joseph-w-valdez) [reply sql-join/what-and-when.sql:7]: I might have mistakenly pasted information into the wrong file. I'll go through everything. Sorry.

Comment (joseph-w-valdez) [reply sql-join/what-and-when.sql:7]: for some reason, when I simply fixed the indenting, it provided a result in the terminal
https://i.imgur.com/3L0rRwE.png

Comment (RobertGardner) [reply sql-join/what-and-when.sql:7]: There must have been something besides indenting that made it start working (or it was working before but you didn't notice?). Indenting in SQL doesn't matter.

**Review by RobertGardner — APPROVED** [review]

File: sql-join/what-and-when.sql (hunk 1 above)
Comment (joseph-w-valdez) [reply sql-join/what-and-when.sql:7]: How strange. I'm not sure. At least it's functional now.

**Pull request merged**